CONTRIBUTING.md

```diff
@@ -0,0 +1,206 @@
+
+```
+Compiled with warnings.
+
+[eslint]
+src\components\pages\GraduateDetails.tsx
+  Line 122:9:  Unexpected console statement  no-console
+
+src\components\pages\Lecturers.tsx
+  Line 160:9:  Unexpected console statement  no-console
+
+src\components\pages\auth\Create.tsx
+  Line 92:5:  Unexpected alert  no-alert
+xt-line to the line before.
+
+WARNING in [eslint]
+src\components\pages\GraduateDetails.tsx
+  Line 122:9:  Unexpected console statement  no-console
+
+src\components\pages\Lecturers.tsx
+  Line 160:9:  Unexpected console statement  no-console
+
+src\components\pages\auth\Create.tsx
+  Line 92:5:  Unexpected alert  no-alert
+
+src\components\pages\auth\Login.tsx  Line 123:5:  Unexpected alert  no-alert
+
+src\components\pages\graduates\Graduates.tsx
+  Line 150:9:  Unexpected console statement  no-console
+
+webpack compiled with 1 warning
+No issues found.
+```
+
+After recieving this, you can open [http://localhost:3000/](http://localhost:3000/) in your browser manually.
+
+## How Can You Contribute 📝
+
+### **Reporting Bugs 🐛**
+
+This section guides you through submitting a bug report for this project. Following these guidelines helps maintainers and the community understand your report 📝, reproduce the behavior 💻, and find related reports 🔎.
+
+Before you create an issue, check if its existing or not. When creating one report include as many details as possible. The information you provide will help others to resolve it.
+
+**Note:** If you find a Closed issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+
+**Note:** Before submiting a bug report make sure that you are on the latest commit.
+
+**How to Submit A Good Bug Report**
+
+Bugs are tracked as [GitHub issues](https://guides.github.com/features/issues/). After figuring out the issue, create an issue on that repository and provide all the information, screen shot also if necessary.
+
+Explain the problem and include additional details to help maintainers reproduce the problem:
+
+- **Use a clear and descriptive title for the issue** to identify the problem.
+- **Describe the exact steps which reproduce the problem** in as many details as possible.
+- **Provide specific examples to demonstrate the steps.** Include links to files or GitHub projects, or copy/pasteable snippets, which you use in those examples.
+- **Describe the behavior you observed** after following the steps and point out what exactly is the problem with that behavior.
+- **Explain which behavior you expected** to see instead and why.
+- **Include screenshots and animated GIFs** which show you following the described steps and clearly demonstrate the problem.
+- **If the problem wasn't triggered by a specific action, describe what you were doing before the problem happened** and share more information using the guidelines below.
```

Comment:
[Note]
this is good for now, but we will need an issue template for this
The issue opened at #50